client/jetpack-connect/disclaimer.jsx

```diff
@@ -28,18 +29,29 @@ class JetpackConnectDisclaimer extends PureComponent {
 			/>
 		);
 
-		const text = translate(
-			'By connecting your site, you agree to {{detailsLink}}share details{{/detailsLink}} between %(companyName)s and %(siteName)s.',
-			{
-				components: {
-					detailsLink,
-				},
-				args: {
-					companyName,
-					siteName,
-				},
-			}
-		);
+		const text =
+			from === 'my-jetpack'
+				? translate(
+						'By clicking {{strong}}Approve{{/strong}}, you agree to {{detailsLink}}sync your site‘s data{{/detailsLink}} with us.',
+						{
+							components: {
+								strong: <strong />,
+								detailsLink,
+							},
+						}
+				  )
+				: translate(
+						'By connecting your site, you agree to {{detailsLink}}share details{{/detailsLink}} between %(companyName)s and %(siteName)s.',
+						{
+							components: {
+								detailsLink,
+							},
+							args: {
+								companyName,
+								siteName,
+							},
```

Comment:
I think we'll want to add a comment here as well to explain the args

Reply:
Yes, good call, sounds good, will do. 👍